GCRCatalogs/catalog_configs/instance_2145656.yaml

```diff
@@ -0,0 +1,3 @@
+subclass_name: instance_catalog.InstanceCatalog
+header_file: /global/cscratch1/sd/danielsf/example_instance_catalogs/phosim_cat_2145656.txt
```

Comment:
Do we need to move this InstanceCatalog into a project space, so that we are not generally dependent on access to my scratch drive (which, as a scratch drive, will get purged in 12 weeks)?